test/spec/duplexBenchmark.js

```diff
@@ -100,6 +100,34 @@ suite.add('compare operation', {
     }
 });
 
+suite.add('compare operation same but deep objects', {
+    setup: function(){
+        var depth = 1;
+        function shallowObj(){
+            return {
+                shallow: {
+                    firstName: "Tomek",
+                    lastName: "Wytrebowicz",
+                    mobileNumbers: [{
+                        number: "12345"
+                    }, {
+                        number: "45353"
+                    }]
+                }
+            };
+        }
+        var obj = shallowObj();
+        var node = obj;
+        while(depth-- > 0){
+            node.nested = shallowObj();
+            node = node.nested;
+        }
+        var obj2 = obj;
```

Comment:
Ha! Thanks for catch it should be 100, i thought to move conf variable to thetop, so you will not have to get to deep into the code to profile this bench.